coulomb-mks-units/src/main/scala/coulomb/mks/package.scala

```diff
@@ -61,4 +65,14 @@ package object mks {
 
   trait Henry
   implicit val defineUnitHenry = DerivedUnit[Henry, Weber %/ Ampere](abbv = "H")
+
+  trait Siemens
+  implicit val defineUnitSiemens = DerivedUnit[Siemens, (Second %^ 3) %* (Ampere %^ 2) %/ (Kilogram %* (Meter %^ 2))](abbv = "S")
+
+  trait ElectronVolt
```

Comment:
This is worth discussing a bit. There are several units called electron volt referring to the same
thing but in terms of energy (this particular version) or length etc:
https://en.wikipedia.org/wiki/Electronvolt

Reply:
IMO the energy-based one you defined is the "right" one to have as the default.  If we ever added other variations, I'd qualify them. ElectronVoltMass, ElectronVoltDistance, etc.

Out of curiosity, why did you add ElectronVolt to 'mks' instead of physical constants?

Reply:
I wasn't sure where to put it really. I think Wikipedia indicates it is an SI unit. I could move it to wherever you think it is the right place

Reply:
good question.  It's not really an MKS unit, but it's not a physical _constant_ per se either.  I think I'd put it in `physicalconstants.units`